server/src/validators/ItemValidators.ts

```diff
@@ -32,6 +32,11 @@ export const itemCreateValidator: Validator[] = [
     type: "array",
     minArrayLength: 1,
   },
+  {
```

Comment:
It is letting me create an item when sending an empty array, I think each item must hava at least one picture, but I am open to hear your thoughts

Reply:
i added the  data type constraints and it's working as expected.